video/clip.go

```diff
@@ -62,6 +62,25 @@ func getRelevantSegment(allSegments []*m3u8.MediaSegment, playHeadTime float64,
 	return 0, fmt.Errorf("error clipping: did not find a segment that falls within %v seconds", playHeadTime)
 }
 
+func ConvertUnixMillisToSeconds(requestID string, firstSegment *m3u8.MediaSegment, startTimeUnixMillis, endTimeUnixMillis int64) (float64, float64, error) {
+	firstSegProgramDateTimeUTC := firstSegment.ProgramDateTime
+	if firstSegProgramDateTimeUTC.IsZero() {
+		return 0.0, 0.0, fmt.Errorf("error clipping: PROGRAM-DATE-TIME of first segment is not set")
+	}
+	firstSegUnixMillis := firstSegProgramDateTimeUTC.UnixNano() / int64(time.Millisecond)
+
+	startTimeSeconds := float64(startTimeUnixMillis-firstSegUnixMillis) / 1000.0
+	endTimeSeconds := float64(endTimeUnixMillis-firstSegUnixMillis) / 1000.0
+
+	log.Log(requestID, "Clipping timestamps",
```

Comment:
@gioelecerati FYI: I couldn't reproduce the issue we saw with timestamps being affected by timezone. So hopefully this little log lets us easily spot an weirdness after we test on staging.